conf/Makefile.chibios

```diff
@@ -137,6 +138,8 @@ CSRC = $(PORTSRC) \
        $(BOARDSRC) \
        $(CHIBIOS)/os/various/evtimer.c \
        $(CHIBIOS)/os/various/syscalls.c \
+       $(CHIBIOS)/os/various/chprintf.c \
+       $(CHIBIOS)/os/various/shell.c \
        $(CHIBIOS_BOARD_MAIN) \
        $($(TARGET).srcs)
```

Comment:
instead of adding `$($(TARGET).srcs)` to CSRC, add them to `ECSRC` like in the Makefile.chibios-libopencm3
Then you don't need to make symlink to have different filenames...